packages/firestore/index.node.ts

```diff
@@ -20,7 +20,7 @@ import { FirebaseNamespace } from '@firebase/app-types';
 import { name, version } from './package.json';
 import { Firestore, IndexedDbPersistenceProvider } from './src/api/database';
 import { configureForFirebase } from './src/config';
-import { FirebaseFirestore as ExpFirebaseFirestore } from './src/exp/database';
+import { Firestore as ExpFirebaseFirestore } from './src/exp/database';
```

Comment:
I know it's just internal so it doesn't matter that much but for consistency in readability, should remove the `Firebase` from the `ExpFirebaseFirestore` too, right? Saw at least one other occurrence in another file.

Reply:
updated